### ISSUE TITLE
[GlobalStep] Voiceover fails to read help tip content for "Product data" on "Add new product" page.

### ISSUE THREAD

**Issue by gglobalstep** [woothemes/woocommerce#38506]
### Prerequisites

- [ ] I have carried out troubleshooting steps and I believe I have found a bug.
- [ ] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

Voiceover fails to read help tip content for "Product data" on "Add new product" page.

### Expected behavior

Screen reader should read the help tip text.

### Actual behavior

Screen reader does not read the help tip text..

### Steps to reproduce

1.  Create any test site.
2. Install and activate all the required plugins.
3. Install and activate Woocommerce.
4. Navigate to 'Add new' product page.
5. Enable screen reader such as VoiceOver.
6. Navigate to help tip element next to Product type dropdown using the keyboard.
7. Observe that, the screen reader does not read the help tip text.

### Instruction Link:
https://github.com/woocommerce/woocommerce/wiki/Release-Testing-Instructions-WooCommerce-7.8#fix35860-adds-missing-aria-label-and-tabindex-html-attributes-to-the-help-tip-element

### Screenshot:

<img width="659" alt="#38506" src="https://github.com/woocommerce/woocommerce/assets/41110392/9c307537-5a15-44ed-8572-75fefe9b39b2">

### WordPress Environment

Woocommerce Version: WooCommerce 7.8.0-beta.2
WordPress version: 6.2.2, 6.1.3

PC:
macOS 13.2.1
-Safari (Version 16.3)

### Isolating the problem

- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

<details>

`
### WordPress Environment ###

WC Version: 7.8.0
REST API Version: ✔ 7.8.0
WC Blocks Version: ✔ 10.2.1
Action Scheduler Version: ✔ 3.5.4
Log Directory Writable: ✔
WP Version: 6.2.2
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: Apache/2.4.57 (Unix) OpenSSL/1.0.2g
PHP Version: 8.0.28
PHP Post Max Size: 1 GB
PHP Time Limit: 30
PHP Max Input Vars: 5000
cURL Version: 7.47.0
OpenSSL/1.0.2g

SUHOSIN Installed: –
MySQL Version: 5.7.33-0ubuntu0.16.04.1-log
Max Upload Size: 512 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 7.8.0
WC Database Prefix: wp_
Total Database Size: 7.74MB
Database Data Size: 5.97MB
Database Index Size: 1.77MB
wp_woocommerce_sessions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 0.13MB + Index: 0.14MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_actions: Data: 0.06MB + Index: 0.11MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 0.06MB + Index: 0.03MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_options: Data: 4.47MB + Index: 0.08MB + Engine InnoDB
wp_postmeta: Data: 0.20MB + Index: 0.17MB + Engine InnoDB
wp_posts: Data: 0.09MB + Index: 0.06MB + Engine InnoDB
wp_snippets: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_term_relationships: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_usermeta: Data: 0.06MB + Index: 0.03MB + Engine InnoDB
wp_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.06MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.05MB + Index: 0.02MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 0.05MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_product_attributes_lookup: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_download_directories: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB

### Post Type Counts ###

attachment: 24
page: 9
post: 2
product: 19
product_variation: 7
revision: 2
shop_coupon: 1
shop_order: 36
wp_global_styles: 1
wp_template: 1
wp_template_part: 1

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (6) ###

Query Monitor: by John Blackbourn – 3.12.3
Code Snippets: by Code Snippets Pro – 3.4.0
Companion Plugin: by Osk – 1.29
WooCommerce Payments: by Automattic – 5.9.0
WooCommerce: by Automattic – 7.8.0-beta.2-pulls-270
WordPress Beta Tester: by WordPress Upgrade/Install Team – 3.4.1

### Inactive Plugins (4) ###

Akismet Anti-Spam: Spam Protection: by Automattic - Anti Spam Team – 5.1
Hello Dolly: by Matt Mullenweg – 1.7.2
Jetpack: by Automattic – 12.1
WooCommerce Smooth Generator: by Automattic – 1.0.3

### Dropin Plugins (1) ###

db.php: Query Monitor Database Class (Drop-in)

### Settings ###

API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –
Enforce Approved Product Download Directories: ✔
HPOS feature screen enabled: –
HPOS feature enabled: –
Order datastore: WC_Order_Data_Store_CPT
HPOS data sync enabled: –

### WC Pages ###

Shop base: #7 - /shop/
Cart: #91 - /test-cart/
Checkout: #94 - /test-checkout/
My account: #10 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Twenty Twenty-Three
Version: 1.1
Author URL: https://wordpress.org
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: /srv/users/userc5a5c85f/apps/userc5a5c85f/public/wp-content/plugins/woocommerce/packages/woocommerce-blocks/templates/notices/error.php
/srv/users/userc5a5c85f/apps/userc5a5c85f/public/wp-content/plugins/woocommerce/packages/woocommerce-blocks/templates/notices/notice.php
/srv/users/userc5a5c85f/apps/userc5a5c85f/public/wp-content/plugins/woocommerce/packages/woocommerce-blocks/templates/notices/success.php

### Subscriptions ###

WCS_DEBUG: ✔ No
Subscriptions Mode: ✔ Live
Subscriptions Live URL: https://spectacled-barracuda.jurassic.ninja
Subscriptions-core Library Version: 5.7.1
Subscription Statuses: –
WooCommerce Account Connected: ❌ No

### Store Setup ###

Country / State: United States (US) — California

### Payment Gateway Support ###

WooCommerce Payments: products
refunds
multiple_subscriptions
subscription_cancellation
subscription_payment_method_change_admin
subscription_payment_method_change_customer
subscription_payment_method_change
subscription_reactivation
subscription_suspension
subscriptions
gateway_scheduled_payments
tokenization
add_payment_method

Direct bank transfer: products
Check payments: products
Cash on delivery: products

### Admin ###

Enabled Features: activity-panels
analytics
product-block-editor
coupons
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page

Disabled Features: core-profiler
minified-js
new-product-management-experience
product-variation-management
settings
async-product-editor-category-field

Daily Cron: ✔ Next scheduled: 2023-05-30 11:20:25 +00:00
Options: ✔
Notes: 70
Onboarding: completed

### WooCommerce Payments ###

Version: 5.9.0
Connected to WPCOM: Yes
Blog ID: 219558303
Account ID: acct_1ND5bnC89JEsJz09

### Action Scheduler ###

Complete: 130
Oldest: 2023-05-29 11:21:38 +0000
Newest: 2023-05-29 14:10:55 +0000

Failed: 1
Oldest: 2023-05-29 11:22:24 +0000
Newest: 2023-05-29 11:22:24 +0000

Pending: 1
Oldest: 2023-05-30 11:21:38 +0000
Newest: 2023-05-30 11:21:38 +0000

### Status report information ###

Generated at: 2023-05-29 15:17:02 +00:00
`

</details>

**Comment by amauryplayero** [comment]
Hi can I take on this issue? Thanks!

**Comment by octaedro** [comment]
Hi @amauryplayero,

> Hi can I take on this issue?

That would be great! Please reach out to us if you need help or have any questions.
Thank you 🙂

**Comment by qant** [comment]
Wow... this case is so unusual... just imagine some one manage the shop from the book reader and this is only ONE option wich is not working for now... i bet there is a TON of this... (i hope i am wrong)

I suggest you to use the special solution like this: https://reciteme.com/user-guide/#

**Comment by amesplant** [comment]
I will look into this issue this week.

**Comment by amesplant** [comment]
Is this still an issue? This is what I see when I activate **VoiceOver** using **Safari** and tabbing to the tooltip next to Product data. 
![image](https://github.com/user-attachments/assets/e5febed1-649f-4f27-9e4a-6021b66f1e6e)

**Comment by vikrampm1** [comment]
@octaedro can you review the comment above? If this is no longer happening, we suggest closing the issue.

**Comment by octaedro** [comment]
Hi @vikrampm1,
Same as @amesplant, I couldn't reproduce the error on my end.

I'm going to close this issue because the error is no longer present.
